[libunwind] Fix build error because of wrong register size

medismailben opened the pull request:
This patch should fix an libunwind build failure that happens on Chromium's Android bots:

https://github.com/llvm/llvm-project/pull/165066#issuecomment-3521706265

The issue is that we're using the `wzr` register to move the value 0 in x1, but this is the 32-bit register.

To prevent this issue, we use an immediate instead of the zero registers.

dtellenbach reviewed (approved):
LGTM with the nit of using an immediate instead.

medismailben committed:
[libunwind] Fix build error because of wrong register size

This patch should fix an libunwind build failure that happens on
Chromium's Android bots:

https://github.com/llvm/llvm-project/pull/165066#issuecomment-3521706265

The issue is that we're using the `wzr` register to move the value 0 in
x1, but this is the 32-bit register. To prevent this issue, we use an
immediate instead of the zero registers.

Signed-off-by: Med Ismail Bennani <ismail@bennani.ma>
